FounderConsole.cs

```diff
@@ -3,27 +3,40 @@
 using System.Linq;
 using System.Text;
 using System.IO;
+using System.Diagnostics; // used for the Stopwatch Class
+using System.Threading; // used for the Stopwatch Class
 
-		// TODO aumentar poder dos valores = números maiores (double por exemplo - maior e mais preciso).
-		// TODO limite de partículas por ciclo
-		// TODO fazer mutação deletéria com probabilidade fixa
-		// TODO fazer mutação e neutra com probabilidade fixa
-		// TODO fazer mutações com probabilidades aleatórias
-		// TODO *** avaliar quando encerrar um paciente e passar para o próximo (média das classes for constante)
-		// TODO fazer mais de um paciente
-		// TODO fazer simulação definindo em quais ciclos ocorrem infecções e deixar pacientes inicais rodando
-		// TODO interface gráfica (gráficos em tempo real, novas janelas para cada paciente etc)
+// TODO aumentar poder dos valores = números maiores (double por exemplo - maior e mais preciso).
+// TODO limite de partículas por ciclo
+// TODO fazer mutação deletéria com probabilidade fixa
+// TODO fazer mutação e neutra com probabilidade fixa
+// TODO fazer mutações com probabilidades aleatórias
+// TODO *** avaliar quando encerrar um paciente e passar para o próximo (média das classes for constante)
+// TODO fazer mais de um paciente
+// TODO fazer simulação definindo em quais ciclos ocorrem infecções e deixar pacientes inicais rodando
+// TODO interface gráfica (gráficos em tempo real, novas janelas para cada paciente etc)
 
-		// TODO: A INTRO explanation about this program, the main use, the aim, how it works, the output and what to do with.
+// TODO: A INTRO explanation about this program, the main use, the aim, how it works, the output and what to do with.
 
 
 namespace multi_dimensional_array
 {
 	public class Program
 	{
+		// create and start the Stopwatch Class. See: https://msdn.microsoft.com/en-us/library/system.diagnostics.stopwatch
```

Comment:
Adicionei um relógio para contar o tempo que o programa leva para executar. O relógio para depois do `for loop` e imprime o resultado depois de `PrintOutput`